.jazzy.yaml

```diff
@@ -0,0 +1,10 @@
+# You can find build configs and their default values here:
+# https://github.com/realm/jazzy/blob/7e8a61a4f1d18ac155a1325572a24ec86e57549f/lib/jazzy/config.rb#L118
+clean: true
+author: Glia Technologies, Inc
+author_url: https://glia.com
+github_url: https://github.com/salemove/ios-sdk-widgets
+root_url: https://docs.glia.com/glia-mobile/docs/ios-widgets-sdk
```

Comment:
should this url point to docs or to our guides?

Reply:
In jazzy it is described as:
> Absolute URL root where these docs will be stored

Will change the URL to point to the API docs themselves.

Reply:
Fixed in 207120b